benchmark/fluid/fluid_benchmark.py

```diff
@@ -186,6 +182,10 @@ def train(avg_loss, infer_prog, optimizer, train_reader, test_reader, batch_acc,
         print("Pass: %d, Loss: %f" % (pass_id, np.mean(train_losses))),
         # evaluation
         if not args.no_test and batch_acc and not args.use_reader_op:
+            if args.use_inference_transpiler:
+                t = fluid.InferenceTranspiler()
+                t.transpile(infer_prog, place)
```

Comment:
I am curious about why we should do `t.transpile` for `infer_prog` in every iteration?

Reply:
After every iteration, parameters will be updated. And `t.transpiler` will fuse parameters. Thus, we should do `t.transpile` before each test, otherwise, the test accuracy will not be right.

Reply:
I see, thanks for your explanation.
But I don't see those explanations in `inference_transpiler.py` or `inference_support_in_fluid_cn.md`. 
Do you have a plan to add some explanations in the two file?

Reply:
Does https://github.com/PaddlePaddle/Paddle/blob/c318aa5ffa6b962cf06e07eb26bd7255a09fbef9/python/paddle/fluid/transpiler/inference_transpiler.py#L22-L40 explain your questions?

Reply:
@chengduoZH I will enhance the explanation of `inference_transpiler.py` in next PR.